src/internal/EnhancedButton.js

```diff
@@ -273,6 +273,8 @@ class EnhancedButton extends React.Component {
       WebkitTapHighlightColor: enhancedButton.tapHighlightColor, // Remove mobile color flashing (deprecated)
       cursor: disabled ? 'default' : 'pointer',
       textDecoration: 'none',
+      margin: 0,
+      padding: 0,
```

Comment:
I'm wondering about the side effect of this change. I will try each page :grin:.